### PR TITLE
Hide duplicated posts in sidebar

### PR DESCRIPTION
**Pull request by LukeHong**
This PR...

## Changes / fixes

- No longer list duplicated posts in both the recent posts list and the featured list
- Fix currently viewing post won't be listed in the sidebar lists issue (#19)

## Screenshots (if applicable)

PR

## Checklist

_Ensure you have checked off the following before submitting your PR._

- [v] tested locally with the [latest release of Hugo](https://github.com/gohugoio/hugo/releases). This requirement is [a standard](https://github.com/gohugoio/hugoThemes#theme-maintenance)
- [v] added new dependencies
- [v] updated the [docs]() ⚠️

### CODE REVIEW

**Comment by onweru** [comment]
@LukeHong, can you confirm that it's consistent with #19?

**Comment by LukeHong** [comment]
@onweru I fixed the issue in the new commit

**Comment by rootwork** [comment]
[`symdiff`](https://gohugo.io/functions/symdiff/) is a Hugo function I hadn't tried before, but it looks nice. I haven't tested this locally yet but going ahead and approving the test builds for now.

**Review by onweru — APPROVED** [review]

**Comment by LukeHong** [comment]
Fix missing signoff issue, should be all good after being approved

**Review by rootwork — APPROVED** [review]
Confirmed that this works as expected.

**Comment by rootwork** [comment]
Everything looks good, going to merge. Thanks for contributing @LukeHong!

**Pull request merged**
